nodewatcher/core/api/serializers.py

```diff
@@ -18,12 +18,22 @@
 
 
 class JSONLDSerializerMixin(object):
+    """
+    Mixin, which adds JSON-LD metadata to a serializer.
+    """
+
     def to_representation(self, instance):
         data = super(JSONLDSerializerMixin, self).to_representation(instance)
 
         # Fix primary key field.
+        id_field = None
         if instance._meta.pk.name in data:
-            del data[instance._meta.pk.name]
+            id_field = instance._meta.pk.name
```

Comment:
Hm, if we expose primary key as `@id`, how can you then know over which field you can filter and sort? Because you probably still have to write `uuid` to filter, and not `@id`?